http-tests/src/test/scala/org/apache/pekko/http/scaladsl/server/directives/FileAndResourceDirectivesSymlinkSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package org.apache.pekko.http.scaladsl.server
+package directives
+
+import java.io.File
+import java.nio.file.{ Files, Paths }
+
+import scala.concurrent.duration._
+
+import org.apache.pekko
+import pekko.http.scaladsl.testkit.RouteTestTimeout
+import pekko.testkit._
+
+import org.scalatest.{ BeforeAndAfterAll, Inside, Inspectors }
+
+class FileAndResourceDirectivesSymlinkSpec extends RoutingSpec
+    with Inspectors with Inside with BeforeAndAfterAll {
+
+  // need to serve from the src directory, when sbt copies the resource directory over to the
+  // target directory it will resolve symlinks in the process
+  val testRoot = new File("http-tests/src/test/resources")
```

Comment:
Better with Files.path, and yes Winodws is using different separator

Reply:
So TIL that Windows has symlinks but only for [Windows 10/11](https://www.howtogeek.com/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/). I don't know how this factors into everything, my initial inclination is to disable the test for Windows since this seems to only really be testing behaviour specific to Posix systems.

@He-Pin Can you confirm if the test passes on your machine (assuming all of the Windows related path separator issues have been solved)?

Reply:
This line is copied from an existing test. The test either works or fails on Windows - either way noone is complaining about it.

Reply:
My understanding is that Windows Java installs know how to apply the file path even if it doesn't use Windows idioms.

Reply:
@He-Pin could you try running `sbt http-tests/test`? It doesn't take that long compared to similar commands in main pekko repo.

Reply:
@He-Pin I created https://github.com/apache/incubator-pekko-http/issues/446 - is ok to say that you are not blocking this? I would like to get this merged so that I can do an RC2 for the Pekko HTTP 1.0.1 release.